mender-convert-package

```diff
@@ -150,7 +150,9 @@ temp_img_name=$(basename "$disk_image")
 
 # Strip the image suffix from temp_img_name and set the original image name as
 # the output image name
-image_name="${temp_img_name%.*}-${device_type}-mender"
+# replaces whitespace and commas with + to seperate device types. 
+# Makes it possible to create files with device_type in the filename.
+image_name="${temp_img_name%.*}-$(sed -e 's|[, ] \?|+|g' <<< $device_type)-mender"
```

Comment:
Suggestion:
I think I would actually prefer to do this in two invocations:
1. Keep the original `image_name` transformation (stripping the suffix).
2. Add the device-types.

Then the comments are more true to what's going on.